bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormStructureParserImpl.java

```diff
@@ -26,18 +26,25 @@
     adapters = FormStructureParser.class)
 public class FormStructureParserImpl implements FormStructureParser {
 
+    private static final String FORM_CONTAINER_RESOURCE_TYPE = "core/fd/components/form/container/v2/container";
```

Comment:
We have a separate constants file for resource types, please re-use it from there.